### PR TITLE
Add pluralization feature to directive

### PR DESCRIPTION
**Pull request by SirLamer**
This adds pluralization support to the v-t direction using the additional property "choice". The implementation is slightly sloppy but lightweight in that it makes use of the existing division between $t and $tc. The main reason I wanted this was to take advantage of the performance optimization opportunity afforded by the directive use case.

I have included unit tests but I have not updated the documentation since I am not certain of your workflow structure you're using for docs.

Sorry I committed the dist file before seeing that your contributing guidelines ask that it not be committed, as a tip for the future I suggest you add the dist folder to .gitignore and have GitHub compile the dist folder for you. In this case you'll probably have to merge the PR and then run a recompile and merge that in to capture any other changes since my fork on March 4th.

### CODE REVIEW

**Comment by SirLamer** [comment]
btw I think this will fail 3 unit tests however those same three tests failed in the fork I took, ie. those tests are not related to the scope of this PR.

**Comment by SirLamer** [comment]
OK done

**Comment by kazupon** [comment]
Thanks!
In additionaly, I hope to revert package - lock.json changing. 🙏

**Comment by SirLamer** [comment]
Yeah I did that already

File: package-lock.json [hunk 1]
```diff
@@ -1,19 +1,9 @@
 {
   "name": "vue-i18n",
-  "version": "7.4.2",
```
Comment (kazupon) [inline package-lock.json]: Please revert package-lock.json changing.  🙏

**Pull request merged**